tock/employees/models.py

```diff
@@ -180,6 +179,11 @@ def is_late(self):
         else:
             return False
 
+    @cached_property
```

Comment:
I could be wrong, but I think you may have to import cached_property.

Reply:
it was already imported! `billable expectations` is also cached.